dts/bindings/timer/nordic,nrf-timer.yaml

```diff
@@ -24,10 +24,20 @@ properties:
   interrupts:
     required: true
 
+  max-frequency:
+    type: int
+    default: 16
+    description: |
+        Maximum timer frequency in MHz.
```

Comment:
What's the benefit of having it in MHz? Couldn't we use just Hz, like for example in SPI: https://github.com/zephyrproject-rtos/zephyr/blob/b116bc7da03ce6a05ce189a7bc36a45c397d5fff/dts/bindings/spi/nordic%2Cnrf-spi-common.yaml#L18-L23 or for `clock-frequency` in several bindings?

Reply:
Changed to Hz